src/condor_ce_router_defaults

```diff
@@ -50,15 +50,23 @@ JOB_ROUTER_DEFAULTS_GENERATED = \\
     set_requirements = True; \\
     /* Note default memory request of 2GB */ \\
     /* Note yet another nested condition allow pass attributes (maxMemory,xcount,jobtype,queue) via gWMS Factory described within ClassAd */ \\
-    eval_set_RequestMemory = ifThenElse(maxMemory isnt undefined, maxMemory, ifThenElse(default_maxMemory isnt undefined, default_maxMemory, 2000)); \\
+    eval_set_OriginalMemory = ifThenElse(maxMemory isnt undefined, maxMemory, ifThenElse(default_maxMemory isnt undefined, default_maxMemory, 2000)); \\
+    set_JOB_GLIDEIN_Memory = "$$(TotalMemory:0)"; \\
+    set_JobMemory = JobIsRunning ? int(MATCH_EXP_JOB_GLIDEIN_Memory)*95/100 : OriginalMemory; \\
+    set_RequestMemory = ifThenElse(WholeNodeWanted is true, !isUndefined(TotalMemory) ? TotalMemory*95/100 : JobMemory, OriginalMemory); \\
     eval_set_remote_queue = ifThenElse(batch_queue isnt undefined, batch_queue, ifThenElse(queue isnt undefined, queue, ifThenElse(default_queue isnt undefined, default_queue, ""))); \\
     /* HTCondor uses RequestCpus; blahp uses SMPGranularity and NodeNumber.  Default is 1 core. */ \\
     copy_RequestCpus = "orig_RequestCpus"; \\
-    eval_set_RequestCpus = ifThenElse(xcount isnt undefined, xcount, \\
+    eval_set_OriginalCpus = ifThenElse(xcount isnt undefined, xcount, \\
         ifThenElse(orig_RequestCpus isnt undefined, \\
             ifThenElse(orig_RequestCpus > 1, orig_RequestCpus, \\
                 ifThenElse(default_xcount isnt undefined, default_xcount, 1)), \\
             ifThenElse(default_xcount isnt undefined, default_xcount, 1))); \\
+    set_GlideinCpusIsGood = !isUndefined(MATCH_EXP_JOB_GLIDEIN_Cpus) && (int(MATCH_EXP_JOB_GLIDEIN_Cpus) isnt error); \\
+    set_JOB_GLIDEIN_Cpus = "$$(TotalCpus:0)"; \\
```

Comment:
What's the point of this and `JOB_GLIDEIN_Memory`? They don't appear to be used in any other expressions